impl/src/templates/partials/build_script.template

```diff
@@ -18,7 +18,11 @@ cargo_build_script(
       {%- for dependency in crate.default_deps.build_dependencies %}
         "{{dependency.buildable_target}}",
       {%- endfor %}
-    ],
+    ]
+    {%- if crate.targeted_deps %} 
+    {%- include "templates/partials/targeted_build_dependencies.template" -%},
```

Comment:
I think I weakly prefer the contents of this template to just be inlined here, unless you foresee another possible usage of it. Open to veto from @UebelAndre on this.

Reply:
Yeah, I'd say either inline this or update `targeted_dependencies.template` to handle this case as well.

Inlining is probably the cleaner option (but I never was that great with jinja style templating)